src/lib/transports/splunkHEC.js

```diff
@@ -31,7 +32,7 @@ class SplunkHEC extends winston.Transport {
 			'event': formattedMessage
 		};
 
-		return fetch('https://http-inputs-financialtimes.splunkcloud.com/services/collector/event', {
+		return (global.fetch || nodeFetch)('https://http-inputs-financialtimes.splunkcloud.com/services/collector/event', {
```

Comment:
Is this right?:
- the imported `nodeFetch` is the fallback and can be used server-side only.
- `global.fetch` will be available client-side (using the Fetch API), and is available globally by default.

Reply:
> global.fetch will be available client-side 

Yes, but I'm not aware this package works on the client-side, there is no mention of it and no Bower configuration which might indicate as such.

> the imported nodeFetch is the fallback and can be used server-side only.

Yep, it's Node.js package.

Reply:
Follow up... this package does not work in the browser, it relies on Node's `https` module and `process.env`